llap-server/src/java/org/apache/hadoop/hive/llap/daemon/impl/ContainerRunnerImpl.java

```diff
@@ -678,7 +678,7 @@ public BlockingQueue<UserGroupInformation> call() throws Exception {
               ugi.addToken(appToken);
               BlockingQueue<UserGroupInformation> queue = new LinkedBlockingQueue<>(numExecutors);
               queue.add(ugi);
-              LOG.debug("Added new ugi pool for " + appTokenIdentifier + ", Pool Size: ");
+              LOG.debug("Added new ugi pool for {} Pool Size:{}", appTokenIdentifier, ugiPool.size());
```

Comment:
Would be better if you could add .(full stop) between {} and Pool Size to make it readable.
**Added new ugi pool for {}. Pool Size:{}**